lib/modules/sections.mjs

```diff
@@ -21,7 +21,7 @@ export async function validateAbstractSection (doc, { mode = MODES.NORMAL } = {}
         result.push(new ValidationError('MISSING_ABSTRACT_SECTION', 'The abstract section is missing.', {
           ref: 'https://authors.ietf.org/required-content#abstract'
         }))
-      } else if (!doc.data.content.abstract || doc.data.content.abstract.length < 1) {
+      } else if (!doc.data.content.abstract || doc.data.content.abstract.length < 2) {
```

Comment:
the scattering of <1 becoming <2 suggest a helper with a name that explains what's going on might be better?